tests/test-basic-user-only.sh

```diff
@@ -21,5 +21,7 @@ set -euo pipefail
 
 . $(dirname $0)/libtest.sh
 
+skip_without_user_xattrs
```

Comment:
OK, let's fix that rather than skip, I'd like to keep as broad test coverage of flatpak as possible.